pkgs/applications/misc/authy/default.nix

```diff
@@ -11,8 +33,8 @@ in
 
 stdenv.mkDerivation rec {
   pname = "authy";
-  version = "1.9.0";
-  rev = "7";
+  version = "2.1.0";
+  rev = "9"; # curl 'https://api.snapcraft.io/api/v1/snaps/search?q=authy' | jq
```

Comment:
maybe it would be helpful to add this command instead above the `version` field? So it's directly the API for `authy` rather than a search query for it.
```
$ curl -H 'X-Ubuntu-Series: 16' 'https://api.snapcraft.io/api/v1/snaps/details/authy?channel=stable' | jq '.download_url,.version'
```

Reply:
Ahh this is convenient. Will update.